jest/jest-base.config.js

```diff
@@ -1,18 +1,34 @@
-const { join } = require('path');
+const { join, dirname } = require('path');
+const regexpEscape = require('escape-string-regexp');
 
-const testPathIgnorePatterns = ['\\.browser-test\\.(js|jsx|ts|tsx)$'];
+const root = dirname(require.resolve('asap-hub/package.json'), '..');
+
+const testPathIgnorePatterns = [
+  '\\.build-output-test\\.(js|jsx|ts|tsx)$',
+  '\\.browser-test\\.(js|jsx|ts|tsx)$',
+];
 
 module.exports = {
   testRunner: require.resolve('jest-circus/runner'),
   testEnvironment: 'node',
   setupFilesAfterEnv: [require.resolve('./flags-setup-after-env.js')],
 
-  cacheDirectory: join(__dirname, '..', '.jest-cache'),
+  cacheDirectory: join(root, '.jest-cache'),
+
+  transform: {
+    '^.+\\.[jt]sx?$': [
+      'babel-jest',
+      { configFile: require.resolve('../babel-cjs.config.js') },
+    ],
+  },
 
+  moduleNameMapper: {
+    '^@asap-hub/([^/]+)$': '@asap-hub/$1/src',
+  },
   modulePathIgnorePatterns: [
-    '<rootDir>/build([^/]*)/',
```

Comment:
Not sure if these ever worked properly since the root dir is different in every project. This should be more stable